samples/csharp/getting-started/DatabaseIntegration/DatabaseIntegration/DatabaseIntegration.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
I think if we go with EF Core 3.0 previews (not a bad idea), we should switch to netcoreapp3.0 as well. Right now things may work, but in an upcoming preview this combination will stop working.

Reply:
Does going with .netcoreapp3.0 force users to use VS2019? I would prefer to have an update like that happen as a change across all projects in the sample. I will add the EFcore 3.0 reference though. 

---
In reply to: [278283185](https://github.com/dotnet/machinelearning-samples/pull/295#discussion_r278283185) [](ancestors = 278283185)